app/enterprise.go

```diff
@@ -35,9 +36,9 @@ func RegisterDataRetentionInterface(f func(*App) einterfaces.DataRetentionInterf
 	dataRetentionInterface = f
 }
 
-var elasticsearchInterface func(*App) einterfaces.ElasticsearchInterface
+var elasticsearchInterface func(*App) searchengine.SearchEngineInterface
```

Comment:
We can rename this to just `searchInterface` since it's not exported.

Reply:
It is not a "searchInterface" is concretely the elasticsearch interface, because this function is initialized by the enterprise code to allow to register elasticsearch. If you compile this without the enterprise code, it is nil, so you don't have ElasticSearch support.